code/object/objectshield.cpp

```diff
@@ -17,34 +17,69 @@
 
 #include <limits.h>
 
+// Private variables
+static double factor = 1.0 / (log(50.0) - log(1.0));	// Factor used in Goober5000's scale_quad
```

Comment:
I think "factor" is a bit too generic a name, especially if it is used as a global variable in this code. Renaming it to "shield_scale_factor" or something like that would make the intent clearer.